src/dawn/Support/IndexGenerator.h

```diff
@@ -0,0 +1,51 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
is there a reason we need this on a per-do-method basis and cannot just reuse the one in stencilInstatntiaon:
https://github.com/MeteoSwiss-APN/dawn/blob/d3db46bf40df399374b13dff0cbdb3d0dc9a2322/src/dawn/Optimizer/StencilInstantiation.h#L501

Reply:
well the reason would be, why would we need to have StencilInstantation object everywhere in the global scope, if we just need an uuid? I would say that StencilInstantiation should use this, rather than owning its own

Reply:
allright